backend/src/main/java/hu/flowacademy/vajdasagbrand/controller/UserController.java

```diff
@@ -64,9 +66,11 @@ public void userRegistration(@RequestBody UserDTO userDTO) throws ValidationExce
 
     @RolesAllowed("SuperAdmin")
     @GetMapping("/getUsers")
-    public Page<User> getUsers(@RequestParam(value = "order_by", required = false) Optional<String> orderBy,
+    public Page<User> getUsers(Authentication authentication,
```

Comment:
Itt mi történik?

Reply:
Ha eddig is fordult, akkor ez unused paraméter, nem?

Reply:
A teszteléshez kellett, nincs rá szükség.
Kikerül